src/main/java/com/google/api/gax/grpc/ServiceApiSettings.java

```diff
@@ -262,5 +263,28 @@ public ServiceApiSettings build() throws IOException {
                                     shouldAutoCloseChannel(),
                                     getOrBuildExecutor());
     }
+
+    private RetrySettings.Builder mergeRetrySettings(RetrySettings.Builder settingsBuilder,
```

Comment:
I think this should move to RetrySettings - it requires knowledge of the full list of attributes, so it would be more maintainable next to those attributes.